.evergreen/auth_oidc/setup_oidc.js

```diff
@@ -10,6 +10,10 @@ admin.runCommand({createRole: 'test1/readWrite', roles:[{role: 'readWrite', db:
 admin.runCommand({createRole: 'test2/read', roles:[{role: 'read', db: 'test'}], privileges: []});
 
 // Wait for the node to be primary.
-while (admin.runCommand('replSetGetStatus').myState != 1) {}
+while (admin.runCommand('replSetGetStatus').myState != 1) {
+    sleep(50);
+}
+
+exit();
```

Comment:
Is the `exit()` intentional?

Reply:
Yes, otherwise it was hanging for several seconds before exiting.